src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -161,6 +161,11 @@ protected enum RegionPlace { Before, Inside, After };
             bool trackExceptions = false)
         {
             Debug.Assert(node != null);
+            var equalsValue = node as BoundEqualsValue;
+            if (equalsValue != null)
+            {
+                node = equalsValue.Value;
```

Comment:
This is not the right way to handle BoundEqualsValue node,the class should simply override VisitEqualsValue method.